python/config/bradley.py

```diff
@@ -0,0 +1,45 @@
+def match_containers(alma_items: list, aspace_containers: list, logger) -> tuple:
+    unmatched_alma_items = []
+    matched_aspace_containers = []
+    for item in alma_items:
+        item_id = item.get("item_data").get("pid")
```

Comment:
Similar to the ASpace loop below, I think this could be a method which returns a dictionary of Alma data, keyed on a tuple of data needed by this config for matching ASpace data, something like:
```
def _get_alma_match_data(list) -> dict[tuple]:
    # for each item
    # create "matchable" (tuple) with the ordered elements which are needed to match aspace
```